fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/event/LootTableLoadingCallback.java

```diff
@@ -45,5 +45,6 @@
 			}
 	);
 
+	// TODO: What should I do about FabricLootSupplierBuilder here?
```

Comment:
New event sadly :(

Reply:
Yeah, I don't think there's another way. I'll make the new event be in the v1 root package and register this event's invoker as a callback for it.